python/tests/test_ld_matrix.py

```diff
@@ -244,6 +244,41 @@ def norm_hap_weighted(
         result[k] = hap_weights[0, k] / n
 
 
+def norm_hap_weighted_ij(
+    result_dim: int,
+    hap_weights: np.ndarray,
+    n_a: int,
+    n_b: int,
+    result: np.ndarray,
+    params: Dict[str, Any],
+) -> None:
+    """
+    Create a vector of normalizing coefficients, length of the number of
+    index tuples. Each allele's statistic will be weighted by the average
+    of the proportion of AB haplotypes in each population present in the
+    index tuple.
+
+    :param result_dim: Number of dimensions in output. Dependent on arity of stat.
+    :param hap_weights: Proportion of each two-locus haplotype.
+    :param n_a: Number of alleles at the A locus.
+    :param n_b: Number of alleles at the B locus.
+    :param result: Result vector to store the normalizing coefficients in.
+    :param params: Params of summary function.
+    """
+    del n_a, n_b  # handle unused params
+    sample_set_sizes = params["sample_set_sizes"]
+    set_indexes = params["set_indexes"]
+
+    for k in range(result_dim):
+        i = set_indexes[k][0]
+        j = set_indexes[k][1]
+        ni = sample_set_sizes[i]
+        nj = sample_set_sizes[j]
+        wAB_i = hap_weights[0, i]
+        wAB_j = hap_weights[0, j]
+        result[k] = (wAB_i / ni / 2) + (wAB_j / nj / 2)
```

Comment:
Based on our discussion earlier this week, I think it's more natural to norm by the total weights and sample sizes, rather than the average. (So: `(wAB_i + w_AB_j) / (ni + nj)`)